Tutorial example/reader.cpp

```diff
@@ -9,31 +9,54 @@ void ListPeople(const tutorial::AddressBook& address_book) {
   for (int i = 0; i < address_book.people_size(); i++) {
     const tutorial::Person& person = address_book.people(i);
 
-    cout << "Person ID: " << person.id() << endl;
-    cout << "  Name: " << person.name() << endl;
+    cout << "  Person ID: " << person.id() << endl;
+    cout << "    Name: " << person.name() << endl;
     if (person.has_email()) {
-      cout << "  E-mail address: " << person.email() << endl;
+      cout << "    E-mail address: " << person.email() << endl;
     }
 
     for (int j = 0; j < person.phones_size(); j++) {
       const tutorial::Person::PhoneNumber& phone_number = person.phones(j);
 
       switch (phone_number.type()) {
         case tutorial::Person::PHONE_TYPE_MOBILE:
-          cout << "  Mobile phone #: ";
+          cout << "    Mobile phone #: ";
           break;
         case tutorial::Person::PHONE_TYPE_HOME:
-          cout << "  Home phone #: ";
+          cout << "    Home phone #: ";
           break;
         case tutorial::Person::PHONE_TYPE_WORK:
-          cout << "  Work phone #: ";
+          cout << "    Work phone #: ";
           break;
       }
       cout << phone_number.number() << endl;
     }
   }
 }
 
+bool ParseDelimitedMessage(tutorial::AddressBook& message, istream& input) {
```

Comment:
Rekomenduočiau perkelti šitą funkciją į atskirą failą, kad ją galėtume naudoti ir kitose programose. vietoj `AddressBook` gali naudoti template (https://www.geeksforgeeks.org/templates-cpp/), kad ji veiktų su visais protocol bufferio tipais.